prow/github/types.go

```diff
@@ -196,7 +197,10 @@ type PushEvent struct {
 	Ref     string   `json:"ref"`
 	Before  string   `json:"before"`
 	After   string   `json:"after"`
+	Compare string   `json:"compare"`
 	Commits []Commit `json:"commits"`
+	Pusher  User     `json:"pusher"`
```

Comment:
Can you add comments on the new fields that you are adding? We need to fill out the rest too but not here.